src/uct/api/uct.h

```diff
@@ -106,6 +106,90 @@ BEGIN_C_DECLS
  * @}
  */
 
+/**
+ * @defgroup UCT_CLIENT_SERVER   UCT client-server operations
+ * @ingroup UCT_API
+ * @{
+ * Defines client-server operations.
+ * The client-server API allows the connection establishment between an active
+ * side - a client, and its peer - the passive side - a server.
+ * The connection can be established through a UCT transport that supports
+ * listening and connecting via IP address and port (listening can also be on INADDR_ANY).
+ *
+ * The following is a general overview of the operations on the server side:
+ *
+ * Connecting:
+ * @ref uct_cm_open
+ *      Open a connection manager.
+ * @ref uct_listener_create
+ *      Create a listener on the CM and start listening on a given IP,port / INADDR_ANY.
+ * @ref uct_listener_conn_request_callback_t
+ *      This callback is invoked by the UCT transport to handle an incoming connection
+ *      request from a client.
+ *      Accept or reject the client's connection request.
+ * @ref uct_ep_create
+ *      Connect to the client by creating an endpoint in case of accepting its request.
```

Comment:
so, each new client connect request results in a new endpoint ?

Reply:
@shamisp yes.